### PR TITLE
Speedy Images

### PR DESCRIPTION
**Pull request by ghost**
Updates Gatsby (+ plugins) to v4, also uses `StaticImage` for rendering our images, so they are fast

Helpful Links:

- https://www.gatsbyjs.com/plugins/gatsby-plugin-image/
- https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#customizing-the-default-options
- https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/#static-images
- https://sharp.pixelplumbing.com/api-resize#resize

For some reason, gatsby-plugin-image's default method of resizing is something called `attention`, which is some weird algorithm to decide where the image is centered.

### CODE REVIEW

**Comment by ghost** [comment]
@jessicasu77 please leave comments wherever you have questions!

**Review by jzhzhang66 — APPROVED** [review]
Looks speedy to me!!!! 

pop off

File: src/components/sections/team.jsx [hunk 1]
```diff
@@ -1,151 +1,143 @@
...
-const RenderTeamClicked = smallTeam => {
-    return smallTeam.teamMembers.map(teamMember =>
-        RenderIndividaulTeamMember(
-            teamMember.picture,
-            teamMember.name,
-            teamMember.description
-        )
-    );
+// Render clickable buttons to render each small team
+const SmallTeamButtons = ({ setSmallTeamClicked }) => {
```
Comment (jessicasu77) [inline src/components/sections/team.jsx:117]: Is there a difference between passing parameters to functions with {} vs without, 
for example what's the difference between 
const SmallTeamButtons = ({ setSmallTeamClicked }) => { 
and
const RenderSmallTeamButtons = setSmallTeamClicked => {

File: src/styles/home.scss [hunk 2]
```diff
@@ -418,14 +421,8 @@
 // Team
 // ---------------------------------------------------------------------------
 .team__container {
-  height: 800px;
-
   @media (max-width: $xxs) {
-    padding: 0px 15px;
-  }
-
-  @media (min-width: $xl) and (max-width: 1399px) {
```
Comment (jessicasu77) [inline src/styles/home.scss:427]: Was having trouble finding the media query keyword that has the meaning of up to but not including (wanted to do something like max-width: 1400px but not include 1400px) 

Is there a keyword for that?

File: src/components/sections/team.jsx (hunk 1 above)
Comment (ghost) [reply src/components/sections/team.jsx:117]: the difference is the first is being destructured, whereas the second is just being named a parameter. here's code that's equivalent to the above, to illustrate the difference:

```javascript
const f1 = (a1) => {
  const setSmallTeamClicked = a1.setSmallTeamClicked;
  // a1 is an object that looks like this: { setSmallTeamClicked: <some value> }
}
const f2 = (setSmallTeamClicked) => {
  // the argument is just named setSmallTeamClicked
}
```

File: src/styles/home.scss (hunk 2 above)
Comment (ghost) [reply src/styles/home.scss:427]: hmm good question, not sure but you could probably do the inverse? ie move the conditional css outside the condition, and then move the stuff outside inside the condition, and then flip it

Comment (ghost) [reply src/styles/home.scss:427]: cc @jzhzhang66

**Pull request merged**